GTViewController/Classes/Custom Classes/GTFollowupModalView.m

```diff
@@ -115,7 +97,27 @@ - (instancetype)initFromElement:(TBXMLElement *)element withStyle:(GTPageStyle*)
                 currentY += buttonPairView.frame.size.height + 10;
                 
             }else if ([modalComponentElementName isEqual:kName_Thank_You]) {
-                self.thankYouPage = nil;
+                
+                self.thankYouView = [[GTFollowupThankYouView alloc]initFromElement:modalComponentElement
+                                                                         withFrame:self.frame
+                                                                     withPageStyle:style];
+                
+                NSArray *listeners = [[TBXML valueOfAttributeNamed:kAttr_listeners forElement:modalComponentElement] componentsSeparatedByString:@","];
+                
+                for (id listener in listeners) {
+                    NSString *listenerName = listener;
```

Comment:
Looks good. You might want to consider adding whitespace trimming either here or in the registerListener method so people maintaining the xml don't get tripped up by accidentally adding whitespace to the start or end of their lister name.

If you do it in the registerListener method you don't have to repeat the trimming code everywhere you read in a list of  listeners